.github/actions/gen-install-scripts/Dockerfile

```diff
@@ -7,6 +7,7 @@ RUN curl -L https://storage.googleapis.com/kubernetes-release/release/v${KUBECTL
     chmod +x /usr/bin/kubectl
 RUN cd /usr/local/bin &&\
     curl -L https://raw.githubusercontent.com/kubernetes-sigs/kustomize/master/hack/install_kustomize.sh | bash
+    # curl -L https://raw.githubusercontent.com/kubernetes-sigs/kustomize/kustomize/v3.10.0/hack/install_kustomize.sh | bash
```

Comment:
[nit] do we need this line?

Reply:
this was when I had problems with different kustomize versions, but not needed any more